godtossing/Pos/Ssc/GodTossing/Workers.hs

```diff
@@ -419,3 +431,46 @@ waitUntilSend msgTag epoch slMultiplier = do
                 ttwMillisecond
                 msgTag
         delay timeToWait
+
+----------------------------------------------------------------------------
+-- Security check
+----------------------------------------------------------------------------
+
+checkForIgnoredCommitmentsWorker
+    :: forall ctx m.
+       SscMode SscGodTossing ctx m
+    => (WorkerSpec m, OutSpecs)
+checkForIgnoredCommitmentsWorker = localWorker $ do
+    counter <- newTVarIO 0
+    void $ onNewSlot True (checkForIgnoredCommitmentsWorkerImpl counter)
+
+-- This worker checks whether our commitments appear in blocks. This
+-- check is done only if we actually should participate in
```

Comment:
The comment should say what happens if this check fails (i.e. just emits a warning).

Reply:
Ok.